frontend-react/src/components/MessageTracker/WarningsErrors.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+// TODO: move this interface into the resources directory
```

Comment:
Doug had started moving network data interfaces like this into `src/config/endpoints/*.ts` where we store the endpoints they return from. There's a few examples in there already, but I think that Resources folder is heavily rest-hooks related so moving things we still use out of there helps with that cleanup later.